sets/technic/42114-volvo-articulated-hauler/powered-up-remote/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Powered Up Remote Control"
+maintainer:
+    user: "repkovsky"
+    name: "Repkovsky"
+image:
+    local: "42114_88010.jpg"
+description:
+    "Control the Volvo articulated hauler with the Powered Up Remote!"
+video:
+    youtube: "j4ZxOhXygNY"
+building_instructions:
+    external: https://www.lego.com/cdn/product-assets/product.bi.core.pdf/6396116.pdf
+code: "#program"
```

Comment:
This will be the link to the name of the section trhat contains the program, so people can easily find it. So maybe we could move the header, as discussed below.